schemas/context/opportunity-contactmap.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/contactmap",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Opportunity Contact Map",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/opportunity"
+  ],
+  "description": "Defines a map containing a set of business customer opportunity contact roles, keyed on contactId , meaning that more than one contact of each opportunity may be carried.",
+  "definitions": {
+    "contactmap": {
+      "properties": {
+        "xdm:contactmap": {
+          "type": "object",
+          "meta:xdmType": "map",
+          "additionalProperties": {
+            "items": {
```

Comment:
Has this been defined correctly? I expected a `"type": "array"` to be defined before `items`

Reply:
Apart from the syntax for defining the map type in XDM, should we use array here also to cover cases where there are multiple roles played by a contact for a given opportunity/account.